package.json

```diff
@@ -13,18 +12,19 @@
   "sideEffects": false,
   "scripts": {
     "dev": "echo 'Note: dev script has been renamed to start:dev' && yarn run start:dev",
-    "build": "yarn run build-clean && NODE_ENV=production IS_BUILDING_NEXTJS=1 next build src",
+    "build": "yarn run build-clean && NODE_ENV=production IS_BUILDING_NEXTJS=1 next build",
     "build-clean": "rimraf build/app",
```

Comment:
Seems like `build-clean` command is no longer needed, can be removed from this line and the one above.

Reply:
Correct, there is no build folder anymore.
I could add a clean script which remove the .next folder instead, thereby removing builds and cached pages.

Reply:
mmm, if it's necessary, then let's add it, otherwise might not be needed anymore. Not sure, it's your call.

Reply:
Just removed it for now in https://github.com/reactioncommerce/example-storefront/pull/667/commits/c02077a65446dd23c09be7ec925879af8e7401a0.